src/app/page.tsx

```diff
@@ -1,14 +1,20 @@
 import { Fredoka } from "next/font/google";
+import Collections from "@/components/Collections";
 
-const fredoka = Fredoka({ subsets: ["latin"], weight: ["300"] });
+export const fredoka = Fredoka({ subsets: ["latin"], weight: ["300", "400"] });
```

Comment:
You should not export fonts and reuse them manually.
I suggest to follow Next.js example and use it with Tailwind.

`tailwind.config.ts`
```ts
theme: {
    extend: {
      fontFamily: {
        freedoka: ['var(--font-freedoka)'],
        roboto: ['var(--font-roboto)'],
      },
    },
  },
```

`app/layout.tsx`
```tsx
const freedoka = Inter({
  subsets: ['latin'],
  variable: '--font-freedoka',
})

// Same with Roboto
```

`some-component.tsx`
```ts
<div className="font-freedoka" />
```